### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'replace'

### ISSUE THREAD

**Issue by jacebrowning** [tomchristie/flask-api#65]
The example (http://www.flaskapi.org/#example) shows that an empty string can be returned for `DELETE` on a resource:

```
@app.route("/<int:key>/", methods=['GET', 'PUT', 'DELETE'])
def notes_detail(key):
    ...
    elif request.method == 'DELETE':
        notes.pop(key, None)
        return '', status.HTTP_204_NO_CONTENT
```

But with Flask 0.11 and Flask-API 0.6.6 this consistency causes this exception:

```
Traceback (most recent call last):
  File ".../Flask-0.11-py3.5.egg/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File ".../Flask-0.11-py3.5.egg/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File ".../Flask-0.11-py3.5.egg/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File ".../Flask-0.11-py3.5.egg/flask/_compat.py", line 33, in reraise
    raise value
  File ".../Flask-0.11-py3.5.egg/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File ".../Flask-0.11-py3.5.egg/flask/app.py", line 1642, in full_dispatch_request
    response = self.make_response(rv)
  File ".../Flask_API-0.6.6-py3.5.egg/flask_api/app.py", line 58, in make_response
    rv = self.response_class(rv, headers=headers, status=status_or_headers)
  File ".../Flask_API-0.6.6-py3.5.egg/flask_api/response.py", line 17, in __init__
    content = renderer.render(content, media_type, **options)
  File ".../Flask_API-0.6.6-py3.5.egg/flask_api/renderers.py", line 112, in render
    mock_content = html_escape(mock_content)
  File ".../Flask_API-0.6.6-py3.5.egg/flask_api/renderers.py", line 20, in html_escape
    text = text.replace(char, replacement)
AttributeError: 'NoneType' object has no attribute 'replace'
```

**Comment by jacebrowning** [comment]
One workaround is to return an empty dictionary instead:

```
return {}, status.HTTP_204_NO_CONTENT
```

but that will also return a dictionary as part of the response.

**Comment by jacebrowning** [comment]
This issue only seems to happen when deleting resources in a web browser.

**Comment by jacebrowning** [comment]
This is fixed in `Flask-API == 0.6.9`.
